bin/bids-validator

```diff
@@ -10,18 +10,18 @@ var argv = require('yargs')
     .alias('version', 'v')
     .demand(1, 1)
     .boolean('ignoreWarnings')
-    .describe('ignoreWarnings', 'disregard non-critical issues')
+    .describe('ignoreWarnings', 'Disregard non-critical issues')
     .boolean('ignoreNiftiHeaders')
-    .describe('ignoreNiftiHeaders', 'disregard NIfTI header content during validation')
+    .describe('ignoreNiftiHeaders', 'Disregard NIfTI header content during validation')
     .boolean('verbose')
-    .describe('verbose', 'Log more extensive information about issues.')
+    .describe('verbose', 'Log more extensive information about issues')
     .boolean('json')
-    .describe('json', 'Output results as JSON.')
+    .describe('json', 'Output results as JSON')
```

Comment:
Up to here, cosmetic changes for consistency (start description with capital letter, do not end with a dot)